playbooks/kind-integration-test-arm64/run.yaml

```diff
@@ -0,0 +1,49 @@
+---
+- hosts: all
+  become: yes
+  roles:
+    - config-golang
+    - install-docker-ce
+    - andrewrothstein.bazel
+  vars:
+    - go_version: 1.12
+    - bazel_ver: '0.23.1'
+
+  tasks:
+    - name: git required repositories
+      shell:
+        cmd: |
+          go get sigs.k8s.io/kind
+          go get k8s.io/kubeadm
+          go get k8s.io/kubernetes
+          go get k8s.io/test-infra
```

Comment:
seems should be ``go get -u k8s.io/test-infra/kubetest`` according to https://github.com/kubernetes/test-infra/tree/master/kubetest I failed to install it with your command(it didn't got installed since it is a too wide package) and succed with  ``go get -u k8s.io/test-infra/kubetest``